frontend/metrics_program.html

```diff
@@ -1,26 +1,35 @@
+<!-- Generic -->
 <option value="ntrips_mode_confirm" data-sizex="4" data-sizey="4">Number of trips</option>
 <option value="ntrips_commute_mode_confirm" data-sizex="4" data-sizey="4">Number of commute trips</option>
 <option value="ntrips_purpose" data-sizex="4" data-sizey="4">Trip count by purpose</option>
 <option value="ntrips_under10miles_mode_confirm" data-sizex="4" data-sizey="4">Trip count under 10 miles</option>
 <option value="miles_mode_confirm" data-sizex="4" data-sizey="4">Trip miles by mode</option>
 <option value="average_miles_mode_confirm" data-sizex="6" data-sizey="4">Average trip length</option>
+<option value="average_miles_mode_confirm2" data-sizex="6" data-sizey="4">Average trip length (selected by users)</option>
 <option value="ntrips_per_day" data-sizex="6" data-sizey="4">Trip frequency</option>
+<option value="ntrips_per_weekday" data-sizex="6" data-sizey="4">Trip frequency (weekday)</option>
+<option value="ts_emissions_user" data-sizex="8" data-sizey="2">Timeseries of emissions</option>
+<option value="ts_energy_user" data-sizex="8" data-sizey="2">Timeseries of energy</option>
+<option value="ts_emissions_vmt" data-sizex="8" data-sizey="2">Timeseries of emissions per mile</option>
+<option value="ts_users" data-sizex="8" data-sizey="2">Timeseries of active users</option>
+<option value="ts_all_modes" data-sizex="8" data-sizey="2">Timeseries of all mode shares</option>
+<!-- Mode Specific -->
+<option value="ntrips_replaced_mode" data-sizex="4" data-sizey="4">Trip miles by replaced mode</option>
 <option value="ntrips_ebike_purpose" data-sizex="4" data-sizey="4">eBike trip count by purpose</option>
-<option value="ntrips_ebike_replaced_mode" data-sizex="4" data-sizey="4">eBike trip count by replaced mode</option>
 <option value="miles_ebike_replaced_mode" data-sizex="4" data-sizey="4">eBike trip miles by replaced mode</option>
+<option value="average_miles_replaced_mode" data-sizex="4" data-sizey="4">Average Miles for each replaced mode (w/Other)</option>
+<option value="average_miles_replaced_mode2" data-sizex="4" data-sizey="4">Average Miles for each replaced mode</option>
 <option value="ntrips_ebike_per_day" data-sizex="6" data-sizey="4">eBike Trip frequency</option>
+<option value="ntrips_ebike_per_weekday" data-sizex="6" data-sizey="4">eBike Trips per weekday</option>
+<option value="sketch_all_energy_impact" data-sizex="6" data-sizey="4">Sketch of Energy Impact (kWH) by Transport Mode</option>
+<option value="sketch_all_mode_energy_impact" data-sizex="6" data-sizey="4">Sketch of Energy Impact for all confirmed trips</option>
 <option value="sketch_energy_impact_ebike" data-sizex="6" data-sizey="4">Sketch of eBike energy impact</option>
+<option value="sketch_all_mode_CO2impact" data-sizex="6" data-sizey="4">Sketch of all mode emission impact</option>
```

Comment:
<img width="862" alt="image" src="https://user-images.githubusercontent.com/5828311/188229195-d946aafc-7a3c-4dbf-bf01-1e204cdf21c7.png">

Should this be removed?

Reply:
Yes, I am not sure this even makes sense any more given that we are only considering the replaced mode for trips with the "mode of interest", not all confirmed trips.

I'm going to merge with it, though, since it is not wrong just extraneous. We can remove it in a small PR in the future.